load/InputFileset.py

```diff
@@ -400,7 +403,7 @@ def numberUSXFileset(self, processedFileset):
 
 	def batchName(self):
 		if self.typeCode == "text" and len(self.filesetId) < 10:
```

Comment:
I wonder if the check for len(self.filesetId) < 10 is needed.  This was probably for the six character text filesets, which don't exist anymore. 
do something like this for now
if typecode = text
***if len(self.filesetId) < 10
******print ("DEBUG: text filesetid less than 10 characters long: " + self.filesetId)
***return self.textFilesetId()